frontend/src/pages/RegistrationPage/useForm.js

```diff
@@ -23,6 +25,23 @@ const useForm = (callback, validate) => {
 
     setErrors(validate(values));
     setIsSubmitting(true);
+
+    const valuesFormData={
+      username: values.username,
+      email: values.email,
+      password: values.password
+    };
+    // valuesFormData.append("username", values.username)
+    // valuesFormData.append("email", values.email)
+    // valuesFormData.append("password", values.password) 
+
+    try{
+      API.post("/user/register", valuesFormData).then((response)=>{
+        console.log(response.data)
+      })
+    }catch(error){
+      console.log(error);
+    }
```

Comment:
```suggestion
    
      API.post("/user/register", valuesFormData)
        .then((response)=>{
          //some successful notification here
        })
        .catch((error) => {
          //some error handling here
        })
```
you will only use try catch if you are using async await. 
You also need to show some notification that you successfully register or there is some error during registration. You may use a toast for example. 
[Semantic-React-Toast](npmjs.com/package/react-semantic-toasts)